builtin/providers/azurerm/resource_arm_availability_set.go

```diff
@@ -93,6 +102,13 @@ func resourceArmAvailabilitySetCreate(d *schema.ResourceData, meta interface{})
 		Tags: expandTags(tags),
 	}
 
+	if managed == true {
+		n := "Aligned"
```

Comment:
Could you clarify this behaviour? The normal approach is to expose the SKU options on on the resource schema as special behaviour can put us in tricky situations with future API updates.